### DISCUSSION TITLE
watchtower not pulling a specific image

### DISCUSSION THREAD

**Discussion by ramyajena** [containrrr/watchtower#1174] [Q&A] [answered]
Hi community, 
I'm new to watch tower, when i run the watch tower it will pull all running image but when i define certain image it will not pulling image from docker hub. Please help me out.

**Comment by piksel** [comment, marked as answer]
If you provide specific names on the command line, watchtower limits it's scope to individual containers based on their name matching those arguments.
You can use `docker ps` to view your containers and see what name corresponds to what image. When starting new containers you can supply the name using the `--name` argument.
